tests/services/order_validation_service/sample_rules/test_sample_rules.py

```diff
@@ -38,9 +60,9 @@ def test_validate_well_position_format(valid_order: MicrosaltOrder):
     # WHEN validating the well position format
     errors = validate_well_position_format(order=valid_order)
 
-    # THEN an error should be returned
-    assert errors
-
     # THEN the error should concern the invalid well position
     assert isinstance(errors[0], WellFormatError)
     assert errors[0].sample_index == 0
+
+    # THEN an error should be returned
+    assert errors
```

Comment:
This changed order will be a pet peeve of mine 💢 Won't change any functionality though so feel free to keep it. The check can also be removed. It does not add any additional safety.